src/drivers/rc_input/RCInput.cpp

```diff
@@ -58,7 +58,7 @@ RCInput::RCInput(const char *device) :
 	}
 
 	if (device) {
-		strncpy(_device, device, sizeof(_device));
+		strncpy(_device, device, sizeof(_device) - 1);
```

Comment:
@dagar Related?

Reply:
Just found along the way testing Linux (emlid_navio2_default) building a driver that uses the new delay macros.

Outside of arm-none-eabi-gcc we don't track particular toolchain versions exactly.

`arm-linux-gnueabihf-g++ 9.3.0`